app/views/content/life-as-a-teacher/age-groups-and-specialisms/primary/_article.html.erb

```diff
@@ -103,7 +103,16 @@
     <!-- Support with teacher training -->
     <%= render 'content/shared/teaching/ect_support' %>
   </section>
-<%= render CallsToAction::AdviserComponent.new(
-    text: "An adviser with years of teaching experience can help you to become a teacher. Chat by phone, text, or email as little or often as you need.",
-  ) %>
+</div>
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::MailinglistBlockComponent.new %>
+      <%= render CallsToAction::FindBlockComponent.new(
+        text: "Explore primary teacher training courses",
+        link_target: "https://find-teacher-training-courses.service.gov.uk/primary"
+      ) %>
+      <%= render CallsToAction::AdviserBlockComponent.new %>
```

Comment:
I'd suggest moving the Adviser one before the Find promo (but as per my other comment, I'd question whether we need the Find promo!)